yt/visualization/tests/test_mesh_slices.py

```diff
@@ -99,3 +97,25 @@ def test_mesh_slices():
     os.chdir(curdir)
     # clean up
     shutil.rmtree(tmpdir)
+
+def test_perfect_element_intersection():
+    '''
+    This test tests mesh line annotation where a z=0 slice
+    perfectly intersects the top of a hexahedral element with node
+    z-coordinates containing both -0 and +0. Before
+    https://github.com/yt-project/yt/pull/1437 this test falsely
+    yielded three annotation lines, whereas the correct result is four
+    corresponding to the four edges of the top hex face.
+    '''
```

Comment:
Can you convert this docstring into a comment? Nose interprets the first line of a docstring on a test as the "name" of the test. That means that right now this test shows up in the test runner like this:

```
This test tests mesh line annotation where a z=0 slice ... ok (0.1249s)
```

Which I think is less useful (and greppable) than just printing the module and test function name.